### ISSUE TITLE
Many `Module not found` errors while using @payloadcms/next-payload@0.0.22 with pnpm

### ISSUE THREAD

**Issue by klapec** [payloadcms/next-payload#24]
I have created a new Next.js project with `npx create-next-app@latest --experimental-app` and then applied all the steps from https://github.com/payloadcms/next-payload/blob/main/README.md.

The only thing that I've done differently is that I used pnpm (@8.4.0) instead of npm/yarn.

I'm getting a lot of errors while attempting to visit the default `/admin` route:

```bash
next-payload-test on  main [!?] via  v18.13.0 on ☁️  (eu-central-1) took 1m55s 
❯ pnpm dev

> next-payload-test@0.1.0 dev /Volumes/dev/repos/next-payload-test
> next dev

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /Volumes/dev/repos/next-payload-test/.env
warn  - You have enabled experimental features (appDir, outputFileTracingExcludes) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

info  - Thank you for testing `appDir` please leave your feedback at https://nextjs.link/app-feedback
event - compiled client and server successfully in 725 ms (269 modules)
wait  - compiling...
event - compiled client and server successfully in 179 ms (269 modules)
wait  - compiling /(payload)/admin/page (client and server)...
event - compiled client and server successfully in 3.4s (4641 modules)
wait  - compiling /favicon.ico/route (client and server)...
wait  - compiling /api/[collection]/init (client and server)...
wait  - compiling /api/[collection]/me (client and server)...
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js
[21:46:10] INFO (payload): Connected to Mongo server successfully!
[21:46:10] INFO (payload): Starting Payload...
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/me.js
wait  - compiling /api/access (client and server)...
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js
wait  - compiling /api/[collection]/[id] (client and server)...
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js
warn  - ./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
Module not found: Can't resolve 'bson-ext' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/bson.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'kerberos' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve '@mongodb-js/zstd' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'snappy/package.json' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
Module not found: Can't resolve 'aws4' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/deps.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js

./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
Module not found: Can't resolve 'mongodb-client-encryption' in '/Volumes/dev/repos/next-payload-test/node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib'

Import trace for requested module:
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/utils.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/collection.js
./node_modules/.pnpm/mongodb@4.8.1/node_modules/mongodb/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/lib/index.js
./node_modules/.pnpm/mongoose@6.5.0/node_modules/mongoose/index.js
./node_modules/.pnpm/payload@1.7.4_monaco-editor@0.38.0_scheduler@0.23.0_typescript@5.0.4/node_modules/payload/dist/payload.js
./payload/payloadClient.ts
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/middleware/withPayload.js
./node_modules/.pnpm/@payloadcms+next-payload@0.0.22_payload@1.7.4_webpack@4.46.0/node_modules/@payloadcms/next-payload/dist/handlers/[collection]/init.js
```

This is obviously related to the way pnpm handles undeclared dependencies and goes away once I add `shamefully-hoist=true` to `.npmrc`.
Would it be possible - on your end - to include missing dependencies? Is there something you could do to improve the out-of-the-box pnpm support?

**Comment by jmikrut** [comment]
Hey @klapec — let us see what we can do here. I think it should be solvable. 

We might not be able to get to this for a few weeks, but it definitely should be able to be resolved.

Thanks for hanging in there!

**Comment by Vinlock** [comment]
Just experienced this as well. 💀

**Comment by walid-mos** [comment]
Same here, not this amount of missing modules, but same, the most of them coming from mongodb package for me.

**Comment by walid-mos** [comment]
I don't know if it's related, but my build vercel contains those errors, and i got a 404 and 504 error from the admin route app when i try to access it on Vercel.

It happens only once deployed, works great in dev
 
<img width="1548" alt="Capture d’écran 2023-08-15 à 00 51 49" src="https://github.com/payloadcms/next-payload/assets/105246577/6afa3ce3-3e54-40fd-82a5-a66968008910">
<img width="2197" alt="Capture d’écran 2023-08-15 à 00 52 21" src="https://github.com/payloadcms/next-payload/assets/105246577/2b12182d-c627-4238-aa3f-45d416f67be5">

**Comment by walid-mos** [comment]
Nevermind i didnt understood but adding this makes it work like a charm : 


> This is obviously related to the way pnpm handles undeclared dependencies and goes away once I add `shamefully-hoist=true` to `.npmrc`. 

The 404 and 504 are not related visibly.

**Comment by jacobsfletch** [comment]
Hey there @klapec, @Vinlock, and @walid-mos, this repo will soon be archived because it's been merged into the main Payload repo here: https://github.com/payloadcms/payload/tree/main/packages/next. Payload 3.0 ships with native support for Next.js and is currently out in beta here: https://github.com/payloadcms/payload-3.0-demo. I'm going to close this issue now but feel free to keep the conversation going either as a [GitHub Discussion](https://github.com/payloadcms/payload/discussions) or on [Discord](https://discord.com/invite/payload). If think issue persists, feel free to open a new ticket on either the demo repo or the monorepo with the `3.x-beta` label, and be sure to include a reproduction so we can fix it faster.
